src/core/src/CoreMain.py

```diff
@@ -53,23 +53,27 @@ def __init__(self, argv):
             patch_operation_requested = execution_config.operation.lower()
             patch_assessor = container.get('patch_assessor')
             package_manager = container.get('package_manager')
+            configure_patching = container.get('configure_patching')
 
-            # if this is an auto patching installation request, log and disable (if enabled by default) the current auto OS update status. NOTE: log status in a separate file in config settings
-            if execution_config.maintenance_run_id is not None and patch_operation_requested == Constants.INSTALLATION.lower():
-                package_manager.disable_auto_os_update()
+            # # if this is an auto patching installation request, log and disable (if enabled by default) the current auto OS update status. NOTE: log status in a separate file in config settings
+            # if execution_config.maintenance_run_id is not None and patch_operation_requested == Constants.INSTALLATION.lower():
+            #     package_manager.disable_auto_os_update()
 
-            # Assessment happens no matter what
-            patch_assessment_successful = patch_assessor.start_assessment()
-
-            # Patching + additional assessment occurs if the operation is 'Installation'
-            if patch_operation_requested == Constants.INSTALLATION.lower():
-                # setting current operation here, to include patch_installer init within installation actions, ensuring any exceptions during patch_installer init are added in installation summary errors object
-                status_handler.set_current_operation(Constants.INSTALLATION)
-                patch_installer = container.get('patch_installer')
-                patch_installation_successful = patch_installer.start_installation()
-                patch_assessment_successful = False
+            if patch_operation_requested == Constants.CONFIGURE_PATCHING.lower():
+                configure_patching.start_configure_patching()
```

Comment:
configure patching must always happen.
assessment happens if it's not configure patching.
installation happens if it's installation.

Reorder code accordingly - only lines 62 and above need to change prior to the exception handling.

Reply:
Resolved.